### PR TITLE
Atu 84 admin update subscriber details

### PR DESCRIPTION
**Pull request by spartacus72**

### CODE REVIEW

File: INSS.EIIR.Web/Areas/Admin/Controllers/SubscriberController.cs [hunk 1]
```diff
@@ -28,14 +37,89 @@ public IActionResult Index()
         }
 
         [Area(AreaNames.Admin)]
-        [Route(AreaNames.Admin + "/subscriber/{subscriberId}")]
+        [Route(AreaNames.Admin + "/subscriber/{subscriberId}", Name = "SubscriberProfile")]
         [Authorize(Roles = Role.Admin)]
         public async Task<IActionResult> Profile(int subscriberId)
         {
             var subscriber = await _subscriberDataProvider.GetSubscriberByIdAsync($"{subscriberId}");
```
Comment (lombardig-nomensa) [inline INSS.EIIR.Web/Areas/Admin/Controllers/SubscriberController.cs:44]: I thought we agreed new code would use the api's and not the data provider directly ?

**Comment by lombardig-nomensa** [comment]
> I thought we agreed GFDI.Happy to apply the change if that's your desire

No we said we would have tech debt tickets for the existing code base but anything new would use the api's.

**Comment by spartacus72** [comment]
Technically, this code existed before we “agreed” that.  But no bother.  I will change it now

File: INSS.EIIR.Web/appsettings-template.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "iirwebdbContextConnectionString": ""
   },
   "ApiSettings": {
-    "BaseUrl": "",
-    "ApiKey": ""
+    "BaseUrl": "https://func-uksouth-dev-eiir-searchservice.azurewebsites.net/api",
```
Comment (glinsolvency) [inline INSS.EIIR.Web/appsettings-template.json:12]: please remove

**Review by glinsolvency — CHANGES_REQUESTED** [review]

**Review by jcoales-nomensa — APPROVED** [review]

**Review by glinsolvency — APPROVED** [review]

**Review by glinsolvency — APPROVED** [review]

**Pull request merged**